### DISCUSSION TITLE
Client-side encryption

### DISCUSSION THREAD

**Discussion by karolgebert** [structurizr/onpremises#54] [Q&A]
Hi,

I run an on-premises installation from docker image and  I uploaded json with my workspace but I am having troubles with Explore page.

When I open network tab in dev tools I can see that there is an information about client-side encryption, but I didn't set any password and in settings there is Add Client-side encryption button enabled. I am really confused.

![image](https://github.com/structurizr/onpremises/assets/3704837/02e95177-c1d5-468f-98c2-897e2be0f72a)

**Comment by simonbrowndotje** [comment]
I imagine this is something related to the `structurizr.url` property not being set appropriately -> https://structurizr.com/share/18571/documentation#troubleshooting
